seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/task/operation/source/RequestSplitOperation.java

```diff
@@ -46,10 +48,15 @@ public RequestSplitOperation(TaskLocation taskID, TaskLocation enumeratorTaskID)
     @Override
     public void run() throws Exception {
         SeaTunnelServer server = getService();
-        SourceSplitEnumeratorTask<?> task =
+
+        RetryUtils.retryWithException(() -> {
+            SourceSplitEnumeratorTask<?> task =
                 server.getTaskExecutionService().getExecutionContext(enumeratorTaskID.getTaskGroupID())
-                        .getTaskGroup().getTask(enumeratorTaskID.getTaskID());
-        task.requestSplit(taskID.getTaskID());
+                    .getTaskGroup().getTask(enumeratorTaskID.getTaskID());
+            task.requestSplit(taskID.getTaskID());
+            return null;
+        }, new RetryUtils.RetryMaterial(Constant.OPERATION_RETRY_TIME, true,
```

Comment:
What causes a NullPointerException?

Reply:
> What causes a NullPointerException?

Because 
```
SourceSplitEnumeratorTask<?> task =
                server.getTaskExecutionService().getExecutionContext(enumeratorTaskID.getTaskGroupID())
                    .getTaskGroup().getTask(enumeratorTaskID.getTaskID());
```

May return null.